README.md

```diff
@@ -2,6 +2,10 @@
 
 A GTFS Schedule (static) [General Transit Feed Specification (GTFS)](https://gtfs.mobilitydata.org/spec/gtfs-schedule) feed validator
 
+This README contains information for master branch of this project, which is under active development. If you'd like to view documentation for past releases of the project, see:
+* [v1.4.0](https://github.com/MobilityData/gtfs-validator/blob/v1.4.0/README.md)
```

Comment:
Looking more closely at these two READMEs, neither actually have the completely correct information for the given release:
1. v1.4.0 docs say to "Download the latest gtfs-validator JAR (cli or web) file from our Releases page" and use `java -jar gtfs-validator-v1.3.0_cli.jar...`, which won't work
2. v2.0.0 docs say "If you're running a v1.x release JAR file you'll need Java 11, and can follow these instructions. The below instructions are for the master branch, which will be v2.0." And it doesn't actually point to where you can find the v2.0 release JAR. Also, the full command for `java -jar...` isn't mentioned.

Suggestions:
1. Update the RELEASE.md steps to include updating the README to reflect using the release JAR before doing the release so we don't make the same mistake in the future
2. Check out both the above release tags into new branches and edit the docs so they are correct (have the correct JAR link and command-line instructions) and commit and tag those (maybe `v1.4.0-docs` and `v2.0.0-docs` respectively), and point to those tags from here so the docs for how to use those past releases are correct.

Reply:
> Update the RELEASE.md steps to include updating the README to reflect using the release JAR before doing the release so we don't make the same mistake in the future

Done in https://github.com/MobilityData/gtfs-validator/pull/905/commits/33de638eeb809cdd91ea359626350f48d5f45a43

Reply:
> Check out both the above release tags into new branches and edit the docs so they are correct (have the correct JAR link and command-line instructions) and commit and tag those (maybe v1.4.0-docs and v2.0.0-docs respectively), and point to those tags from here so the docs for how to use those past releases are correct.

👍🏾 two new PRs upcoming :)

Reply:
*v2.0.0: https://github.com/MobilityData/gtfs-validator/pull/915
*v1.4.0: https://github.com/MobilityData/gtfs-validator/pull/914

cc @kaiomi-i for more context :)